core-api/src/main/java/com/sequenceiq/distrox/api/v1/distrox/endpoint/DistroXUpgradeV1Endpoint.java

```diff
@@ -51,4 +53,16 @@ DistroXUpgradeV1Response upgradeClusterByNameInternal(@PathParam("name") String
     DistroXUpgradeV1Response upgradeClusterByCrnInternal(@PathParam("crn") String crn, @Valid DistroXUpgradeV1Request distroxUpgradeRequest,
             @QueryParam("initiatorUserCrn") String initiatorUserCrn);
 
+    @POST
+    @Path("{name}/sync_cm")
```

Comment:
is this an upgrade related thing?

Reply:
When an upgrade fails, versions in Component, ClusterComponent table may slip apart from what is on the cluster. Further, if a customer tries to fix the cluster by hand and installs the newer version of a parcel, then the cluster might be ok but that is not reflected in the aformentioned DB tables.

The purpose of this sync_cm command is to sync versions in the cbdb tables to what is really on the CM. So in this regard this is an upgrade related command.